File: Pre_Intervention.py
import numpy as np
from Visualisierung import Visualisierung

np.random.seed(31425926)
N = 500

V = np.random.poisson(5, N)
W = np.random.poisson(5, N)
X = V + np.random.poisson(3, N)
Y = V + W + X + np.random.poisson(1, N)
Z = X + Y + np.random.poisson(1, N)

Visualisierung(X, Z, "X", "Z")
Visualisierung(V, W, "V", "W")
Visualisierung(Y, Z, "Y", "Z")

File: Pre_Intervention.R
source("Visualisierung.R")

set.seed(31415926)

N = 2500

V = rnorm(N, 0, 1)
W = rnorm(N, 0, 2)
X = W + rnorm(N, 0, 1)
Y = W + X + rnorm(N, 0, 0.1)
Z = V + Y + rnorm(N, 0, 0.2)

Visualisierung(V, W)

Visualisierung(X, Y)

Visualisierung(W, Z)
